app/reducers.js

```diff
@@ -15,10 +21,44 @@ const currentQuestions = (state = [], action) => {
 	}
 };
 
+// Retrieve Activities List from server
+const currentActivities = (state = [], action) => {
+	switch (action.type) {
+		case ACTIVITY_LOADED: {
+			const { activityList } = action;
+			return activityList || state;
+		}
+		default:
+			return state;
+	}
+};
+
+const myActivities = (state = [], action) => {
+	var activityIndex = state.indexOf(action.activityId);
+
+	switch (action.type) {
+		case ADD_ACTIVITY: {
+			if (activityIndex < 0){
+				return [...state, action.activityId];
+			}
+			break;
```

Comment:
Would this work?  I think we want to return the default `state` if not returning anything